### DISCUSSION TITLE
子应用引入高德地图，启动主应用的时候报错了，怎么解决？如图

### DISCUSSION THREAD

**Discussion by liweihan123** [jd-opensource/micro-app#651] [Q&A]
![图片](https://user-images.githubusercontent.com/66237051/199480963-3b322bc8-2a8c-42ba-931d-255b3a6ab78c.png)
请问这个是因为什么引起的呢，怎么解决呢

**Comment by bailicangdu** [comment]
把这段js放到基座中引入试试呢

**Reply by liweihan123** [reply]
把这段js放到基座后，去除子应用中对应的js就可以运行了，但是不去除子应用的js的话加载不出子应用同时也不报错，也就是说子应用单独发布的话得把js重新得加回去，是这样吧？

**Reply by bailicangdu** [reply]
是的，子应用的js加上exclude属性，这样子应用单独运行也可以 https://micro-zoe.github.io/micro-app/docs.html#/zh-cn/configure?id=exclude%e8%bf%87%e6%bb%a4%e5%85%83%e7%b4%a0

**Reply by liweihan123** [reply]
可以了，谢谢啦，js放在基座的话子应用是可以用到的是吧！
